### ISSUE TITLE
feature request: webinar recording (multiple input sources)

### ISSUE THREAD

**Issue by edmundlaugasson** [MaartenBaert/ssr#801]
Would it be possible to record a webinar? I mean to record sound card monitor AND microphone (input) simultaneously. E.g. to record Jitsi Meet webinar or similar. For instance in case of Jitsi Meet there are quite often issues with cloud-based recording and implementing [local recording](https://github.com/jitsi/jitsi-meet/issues/6014) seems to be challenging. If SSR could allow to record multiple sources simultaneously then lots of similar issues could be solved.

**Comment by MaartenBaert** [comment]
Yes, see: https://www.maartenbaert.be/simplescreenrecorder/recording-game-audio/#recording-game-audio-and-microphone-at-the-same-time

**Comment by edmundlaugasson** [comment]
Currently it seems that already long time ago is newer version than 4.1 out of PulseAudio ([PulseAudio homepage](https://www.freedesktop.org/wiki/Software/PulseAudio/), [in Ubuntu](https://packages.ubuntu.com/search?keywords=pulseaudio&searchon=names&suite=all&section=all), [in Debian](https://packages.debian.org/search?keywords=pulseaudio&searchon=names&suite=all&section=all), etc). Noticed on the [referred site](https://www.maartenbaert.be/simplescreenrecorder/recording-game-audio/#recording-game-audio-and-microphone-at-the-same-time) the hint about workaround of the [PulseAudio bug](https://bugs.freedesktop.org/show_bug.cgi?id=66424) - would be good to know whether it is fixed since which PulseAudio version or not.

At the same time it doesn't seem very usable - why not just under "Audio input" section create two tabs: on first is current "Record audio"  and on second "Record microphone" and both have backend and source choices like currently audio has. Would be logical and much more usable. Hopefully is there a solution to avoid listening own voice which is really disturbing. If there is really no other way to record both. Possibly own voice echo not disturbing in case of gaming but when speaking more than just few words, it will be very disturbing.

**Comment by MaartenBaert** [comment]
There's not enough UI space at the moment for multiple audio inputs, and the synchronizer currently can't handle more than one audio stream. In other words it requires significant changes to the code. I am planning to do it eventually.

**Comment by edmundlaugasson** [comment]
Thank you for considering! Well - in my humble opinion there is plenty of space - just a small two-tab bottom section won't take more than current UI allows - in next of "Record audio" there is plenty of space for second tab.  When changing tab, also its content will change - so you have in same UI just two tabs overlapping each other as usually tabs do - each tab has own content. I guess tab-like design would fit into current UI pretty well without distraction. But certainly it is up to you, what you decide - just proposing.
